### PR TITLE
release 2.11.0

### PR DESCRIPTION
**Pull request by bmc-msft**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,21 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 2.11.0
+### Changed
+* Agent: Continued log simplificaiton and clarification.  [#736](https://github.com/microsoft/onefuzz/pull/736), [#740](https://github.com/microsoft/onefuzz/pull/740), [#742](https://github.com/microsoft/onefuzz/pull/742)
```
Comment (ranweiler) [inline CHANGELOG.md:9]:
```suggestion
* Agent: Continued log simplification and clarification.  [#736](https://github.com/microsoft/onefuzz/pull/736), [#740](https://github.com/microsoft/onefuzz/pull/740), [#742](https://github.com/microsoft/onefuzz/pull/742)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,21 @@ All notable changes to this project will be documented in this file.
...
+* Agent: Prevent invalid queue messages from being ignored. [#731](https://github.com/microsoft/onefuzz/pull/731)
+* Agent: Separated module and symbol names for Windows debugger-based crash reports. [#723](https://github.com/microsoft/onefuzz/pull/723)
+* Deployment/Agent: Updated AFL++ to 3.11c.  [#728](https://github.com/microsoft/onefuzz/pull/728)
+* CLI/Deployment: Updated Python dependencies.  [#721](https://github.com/microsoft/onefuzz/pull/721)
+* Agent: Updated stack minimization regular expressions from ClusterFuzz.  [#722](https://github.com/microsoft/onefuzz/pull/722)
+* Service: Removed user's identity information from instance logging instances.  [#724](https://github.com/microsoft/onefuzz/pull/724), [#725](https://github.com/microsoft/onefuzz/pull/725)
```
Comment (ranweiler) [inline CHANGELOG.md:15]:
```suggestion
* Service: Removed user's identity information from logging to user instance.  [#724](https://github.com/microsoft/onefuzz/pull/724), [#725](https://github.com/microsoft/onefuzz/pull/725)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,21 @@ All notable changes to this project will be documented in this file.
...
+* Agent: Continued development related to upcoming features. [#699](https://github.com/microsoft/onefuzz/pull/699), [#729](https://github.com/microsoft/onefuzz/pull/729), [#733](https://github.com/microsoft/onefuzz/pull/733), [#735](https://github.com/microsoft/onefuzz/pull/735), [#738](https://github.com/microsoft/onefuzz/pull/738), [#739](https://github.com/microsoft/onefuzz/pull/739)
+
+### Fixed
+* Deployment: Worked around a race condition in service principal creation. [#716](https://github.com/microsoft/onefuzz/pull/716)
+* Agent: Dotfiles are now ignored in libFuzzer related directories.  [#741](https://github.com/microsoft/onefuzz/pull/741)
```
Comment (ranweiler) [inline CHANGELOG.md:20]:
```suggestion
* Agent: Dotfiles are now ignored in libFuzzer-related directories.  [#741](https://github.com/microsoft/onefuzz/pull/741)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -135,7 +150,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * Service: Handle scaleset nodes that never register, such as nodes with instance-specific setup script failures.  [#518](https://github.com/microsoft/onefuzz/pull/518)
 
 ### Changed
-* Agent: Added stdout/stderr logging and clarifying context during failures to the `generic_analysis` task.  [#522](https://github.com/microsoft/onefuzz/pull/522)
+* Agent: Added STDOUT/STDERR logging and clarifying context during failures to the `generic_analysis` task.  [#522](https://github.com/microsoft/onefuzz/pull/522)
```
Comment (ranweiler) [inline CHANGELOG.md:153]: FWIW, I think it's actually better to standardize on lowercase for `stderr`/`stdout`. Rationale: I suspect it's how most people write them, and its how they appear as global vars and in man pages. If you want to flag them as being special sorts of words (i.e. identifiers), backticks for code formatting might be better.

**Review by ranweiler — CHANGES_REQUESTED** [review]

**Review by chkeita — APPROVED** [review]

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,21 @@ All notable changes to this project will be documented in this file.
...
 
+## 2.11.0
+### Changed
+* Agent: Continued log simplification and clarification.  [#736](https://github.com/microsoft/onefuzz/pull/736), [#740](https://github.com/microsoft/onefuzz/pull/740), [#742](https://github.com/microsoft/onefuzz/pull/742)
+* Agent: Prevent invalid queue messages from being ignored. [#731](https://github.com/microsoft/onefuzz/pull/731)
+* Agent: Separated module and symbol names for Windows debugger-based crash reports. [#723](https://github.com/microsoft/onefuzz/pull/723)
+* Deployment/Agent: Updated AFL++ to 3.11c.  [#728](https://github.com/microsoft/onefuzz/pull/728)
+* CLI/Deployment: Updated Python dependencies.  [#721](https://github.com/microsoft/onefuzz/pull/721)
+* Agent: Updated stack minimization regular expressions from ClusterFuzz.  [#722](https://github.com/microsoft/onefuzz/pull/722)
+* Service: Removed user's identity information from logging to user instance.  [#724](https://github.com/microsoft/onefuzz/pull/724), [#725](https://github.com/microsoft/onefuzz/pull/725)
```
Comment (ranweiler) [inline CHANGELOG.md:15]: Oops, I dropped an `s` in my original suggestion:
```suggestion
* Service: Removed user's identity information from logging to user instances.  [#724](https://github.com/microsoft/onefuzz/pull/724), [#725](https://github.com/microsoft/onefuzz/pull/725)
```

**Review by ranweiler — APPROVED** [review]

**Pull request merged**